src/test/java/com/powsybl/openloadflow/ac/GeneratorRemoteControlTest.java

```diff
@@ -395,6 +395,32 @@ void testGeneratorRemoteReactivePowerControl() {
         assertReactivePowerEquals(targetQ, l12.getTerminal(TwoSides.ONE));
     }
 
+    @Test
+    void testGeneratorRemoteReactivePowerControlOnNonImpdantBranch() {
+        // create a basic 4-buses network
+        Network network = FourBusNetworkFactory.createBaseNetwork();
+        Generator g4 = network.getGenerator("g4");
+        Line l34 = network.getLine("l34");
+        l34.setR(0).setX(0);
+
+        double targetQ = 1.0;
+
+        // disable voltage control on g4
+        g4.setTargetQ(0).setVoltageRegulatorOn(false);
+
+        // first test: generator g4 regulates reactive power on line 4->3 (on side of g4)
```

Comment:
Maybe we can remove "first" since there is only one test here (to avoid confusion).